util/globals.go

```diff
@@ -58,4 +63,23 @@ func Init() {
 
 	backendFormatted := logging.NewBackendFormatter(backend, logFormat)
 	logging.SetBackend(backendFormatted)
+
+	// read environment variable of level
+	var logLevel logging.Level
+	switch strings.ToLower(os.Getenv(EnvLogLevel)) {
```

Comment:
Should this be strings.toUpper ?

Reply:
Yeah, your right, overlooked that one. Thanks